hackerRank/a-very-big-sum/a-very-big-sum.test.js

```diff
@@ -0,0 +1,18 @@
+const aVeryBigSum = require('./a-very-big-sum');
+
+describe('Add a Very Big Sum of [1000000001, 1000000002, 1000000003, 1000000004, 1000000005] to return (5000000015)', () => {
```

Comment:
the test description isn't good ,you said add a very big some OF then the numbers ,that isn't so good 

Adds a very big sum maybe

Reply:
Done